Exception in thread "Thread-0" 8 Apr 2024 13:35:49 GMT [ERROR] java.util.ConcurrentModificationException
8 Apr 2024 13:35:49 GMT [ERROR] 	at java.base/java.util.HashMap$HashIterator.nextNode(HashMap.java:1597)
8 Apr 2024 13:35:49 GMT [ERROR] 	at java.base/java.util.HashMap$KeyIterator.next(HashMap.java:1620)
8 Apr 2024 13:35:49 GMT [ERROR] 	at me.redstoner2019.main.serverstuff.ServerMain$2.run(ServerMain.java:235)
8 Apr 2024 13:35:49 GMT [ERROR] 	at java.base/java.lang.Thread.run(Thread.java:833)
